### ISSUE TITLE
Update all logstash 1.5 plugins 

### ISSUE THREAD

**Issue by rparkhunovsky** [elastic/logstash#3993]
I'd like have logstash plugins updated along with logstash 1.5 installation via puppet. I could have used "exec" for that but running something like:
```
./plugin update
You are updating logstash-codec-collectd to a new version 2.0.0, which may not be compatible with 1.0.1. are you sure you want to proceed (Y/N)?
```
stops the flawless execution.
As well it's not clear if it's safe to install 2.x plugins for logstash 1.5 nor it is possible to make it update only feasible versions.
Is it possible to abandon questions and install only appropriate updates?

**Comment by toleksyn** [comment]
+1

**Comment by oshmyrko** [comment]
It will be awesome :pray:

**Comment by jordansissel** [comment]
Good idea! We're still polishing up some of the rougher aspects of plugin management, and I think we'll have this improved soon, though not sure when. I don't think Logstash 1.5 will get this fix, 2.0 should, though.

**Comment by colinsurprenant** [comment]
there are 2 problems here:
1- the fact that we are using a interactive prompt, which we should not, see #3744 
2- the fact that `bin/plugin update` selects the wrong plugin version to update see #4067

**Comment by cladden** [comment]
Is there a manual workaround to install a specific version of the 1.5 plugin?

**Comment by purbon** [comment]
@cladden you can use ```bin/plugin install --version [version] [plugin name]``` to install an specific plugin version.
